src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -141,6 +143,32 @@ private void WriteScopeInformation(Utf8JsonWriter writer, IExternalScopeProvider
             }
         }
 
+        private static void WriteItem(Utf8JsonWriter writer, KeyValuePair<string, object> item)
+        {
```

Comment:
cc: @layomia

Reply:
There should be a null-check for null values; [Utf8JsonWriter.WriteNull](https://docs.microsoft.com/dotnet/api/system.text.json.utf8jsonwriter.writenull?view=netcore-3.1) should be used in those cases.

[Utf8JsonWriter.WriteNumber](https://docs.microsoft.com/dotnet/api/system.text.json.utf8jsonwriter.writenumber?view=netcore-3.1) and [Utf8JsonWriter.WriteBoolean](https://docs.microsoft.comdotnet/api/system.text.json.utf8jsonwriter.writeboolean?view=netcore-3.1) should be used instead of the serializer below.

Using the `WriteX` vs `WritePropertyName` + `WriteXValue` allows for one less method call.

Reply:
Updated as per feedback.